scripts/release.sh

```diff
@@ -118,3 +118,18 @@ npm run build-and-publish
 )
 
 printf "\n\nRelease: Package was published to npm."
+
+for d in packages/react-instantsearch/examples/* ; do 
+    cd $d
+    yarn upgrade react-instantsearch@$newVersion
+    cd ../../../..
+done
+
+commitMessage="chore(deps): update examples to react-instantsearch v$newVersion"
+git add .
+printf "$commitMessage" | git commit --file -
+
+printf "\n\nUpdate of react-instantsearch in all examples: almost done, check everything in another terminal tab if you want.\n"
```

Comment:
Maybe split this two logs so it's easier to get that almost done is not for the react-instantsearch upgrade?